exonum/src/blockchain/tests.rs

```diff
@@ -459,8 +459,11 @@ fn handling_tx_panic_error() {
     ];
 
     for (tx, expected_err) in failed_transactions {
-        let actual_err = execute_transaction(&mut blockchain, tx.sign(TEST_SERVICE_ID, pk, &sk))
-            .expect_err("Transaction must fail");
+        let actual_err = execute_transaction(
+            &mut blockchain,
+            tx.sign(TEST_SERVICE_ID, keys.public_key(), keys.secret_key()),
```

Comment:
Nit: You can refactor `Transaction::sign` to accept `&KeyPair` instead of separate keys.